src/components/Banner/Banner.mdx

```diff
@@ -25,6 +25,7 @@ import { changelogData } from "./bannerChangelogData";
 - {<Link href="#with-html-content" target="_self">With HTML Content</Link>}
 - {<Link href="#custom-color" target="_self">Custom Color</Link>}
 - {<Link href="#dismissible" target="_self">Dismissible</Link>}
+- {<Link href="#string-content-with-html" target="_self">With HTML content passed as string</Link>}
```

Comment:
In order for the link to work, the hash should be `#with-html-content-passed-as-string` that should match the title on line 238.